docs/html/topics/vcs-support.md

```diff
@@ -0,0 +1,163 @@
+# VCS Support
+
+pip supports installing from various version control systems (VCS).
+This support requires a working executable to be available (for the version
+control system being used). It is used through URL prefixes:
+
+- Git -- `git+`
+- Mercurial -- `hg+`
+- Subversion -- `svn+`
+- Bazaar -- `bzr+`
+
+## Supported VCS
+
+### Git
+
+The supported schemes are `git+file`, `git+https`, `git+ssh`, `git+http`,
+`git+git` and `git`. Here are some of the supported forms:
+
+```none
+git+ssh://git.example.com/MyProject#egg=MyProject
+git+file:///home/user/projects/MyProject#egg=MyProject
+git+https://git.example.com/MyProject#egg=MyProject
+```
+
+```{warning}
+The use of `git`, `git+git`, and `git+http` schemes is discouraged.
+The former two use [the Git Protocol], which lacks authentication, and HTTP is
+insecure due to lack of TLS based encryption.
+```
```

Comment:
Maybe. Let's defer this for a follow up PR though, since one of the things I like about the way it's currently written is that all the sections have the exact same start that lists all the things.